public/app/routes/dashboard_loaders.ts

```diff
@@ -4,18 +4,20 @@ import { UrlQueryMap } from '@grafana/runtime';
 import { DashboardLoaderSrv } from 'app/features/dashboard/services/DashboardLoaderSrv';
 import { BackendSrv } from 'app/core/services/backend_srv';
 import { ILocationService } from 'angular';
+import { AppEventEmitter } from './GrafanaCtrl';
+import { dashboardFetchStart } from '@grafana/data';
 
 export class LoadDashboardCtrl {
   /** @ngInject */
   constructor(
-    $scope: any,
+    $scope: AppEventEmitter & { [key: string]: any },
```

Comment:
I think I've seen `{ [key: string]: any }` type a couple of times in this PR, maybe we should go ahead and create a `Scope` type?